packages/oss-console/src/components/Executions/ExecutionDetails/ExecutionNodeURL.tsx

```diff
@@ -31,19 +32,18 @@ export const ExecutionNodeURL: React.FC<{
   const isHttps = /^https:/.test(window.location.href);
   const ref = React.useRef<HTMLDivElement>(null);
 
-  const code = isHttps
-    ? // https snippet
-      `from flytekit.remote.remote import FlyteRemote
+  const config =
+    env.CODE_SNIPPET_USE_AUTO_CONFIG === "true"
+      ? 'Config.auto()'
+      : isHttps
```

Comment:
For context, this follows the nested ternary pattern that I've seen e.g. here https://github.com/flyteorg/flyteconsole/blob/7636e32d2b1dbefb7d08fb79e5ed063c3ad44ca4/packages/oss-console/src/components/flytegraph/ReactFlow/transformDAGToReactFlowV2.tsx#L199-L205

Since python booleans aren't one-to-one with JS booleans, it would have to look like
```js
const insecure = isHttps ? "False" : "True"
 ... `Config.for_endpoint("${window.location.host}",${insecure})`
```
which I wasn't sure would be much clearer.

Reply:
Gotcha! Okay, updated it here: https://github.com/flyteorg/flyteconsole/pull/883/commits/0dfb7b84fd6f9b6ae0d8ed72c5894e856d2e3d20